packages/create-app/templates/default-app/packages/app/src/App.tsx

```diff
@@ -51,7 +57,12 @@ const routes = (
       {entityPage}
     </Route>
     <Route path="/docs" element={<TechdocsPage />} />
-    <Route path="/create" element={<ScaffolderPage />} />
+    <Route path="/create" element={<ScaffolderPage />}>
+      <ScaffolderCustomFields>
+        <RepoUrlPickerFieldExtension />
```

Comment:
Wonder if we actually shouldn't just provide the built-in ones by default, so that there's no need to provide this unless you're actually adding custom ones? And if you do provide custom ones, we could remove the default ones too, so you'd have to explicitly add them back.

Perhaps `ScaffolderCustomFields` -> `ScaffolderFieldExtensions` as well?